Bottom: 0a7ef0434f824def5622eb33307ca44708b3c45c
Top:    d2243a05243b79ca3492733e8928f68f65852c53
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-16 03:38:03 +0100

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8639614..b0f4558 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1674,6 +1674,7 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
 #endif
       /* add edge to polygon */
 //      if ((newn = poly_CreateNodeFull (cur->point, cur->is_round, cur->cx, cur->cy, cur->radius)) == NULL) /* XXX: DIRECTION - might we need to query the previous point for arc details ?? */
+      /* XXX: The cur->next part of the line below is somewhat tested, but was determined by trial and error */
       if ((newn = poly_CreateNodeFull (cur->point, (dir == FORW) ? cur->is_round : cur->next->is_round,
                                                    (dir == FORW) ? cur->cx       : cur->next->cx,
                                                    (dir == FORW) ? cur->cy       : cur->next->cy,
